src/github/prComment.ts

```diff
@@ -287,8 +287,8 @@ export class GHPRComment extends CommentBase {
 
 	get suggestion(): string | undefined {
 		const suggestionBody = this.rawComment.body.match(SUGGESTION_EXPRESSION);
-		if (suggestionBody?.length === 4) {
-			return `${suggestionBody[2]}\n`;
+		if (suggestionBody?.length === 5) {
+			return suggestionBody[3] ? `${suggestionBody[3]}\n` : '';
 		}
 	}
```

Comment:
Btw you can use named capture groups here which are more maintainable than using an index:

```ts
const SUGGESTION_EXPRESSION = /```suggestion(\r\n|\n)((?<suggestion>[\s\S]*?)(\r\n|\n))?```/;

		const match= this.rawComment.body.match(SUGGESTION_EXPRESSION);
		const suggestionBody = suggestionBody?.groups.suggestion;
		if (suggestionBody) {
			return suggestionBody? `${suggestionBody}\n` : '';
		}
```

Reply:
I didn't know about this! Added in https://github.com/microsoft/vscode-pull-request-github/pull/5213.